src/central-server/admin-service/core/src/main/java/org/niis/xroad/cs/admin/core/facade/SignerProxyFacadeMockHttpImpl.java

```diff
@@ -68,12 +71,11 @@ public class SignerProxyFacadeMockHttpImpl implements SignerProxyFacade {
     public SignerProxyFacadeMockHttpImpl(RestTemplateBuilder builder, @Value("${signerProxyMockUri}") String signerProxyMockUri,
                                          ObjectMapper objectMapper) {
         log.info("Creating mocked SignerProxyFacade");
-
+        ConnectionConfig.custom().build();
         final CloseableHttpClient client = HttpClients.custom()
                 .setDefaultRequestConfig(RequestConfig.custom()
-                        .setConnectTimeout(1000)
-                        .setSocketTimeout(1000)
```

Comment:
was leaving out this setSocketTimeout intentional?

Reply:
api removed it, refactored to use newer HC5 api to get it back, let's hope it works as expected